File: LeetCodeProblems-Python/26. Remove Duplicates from Sorted Array.py
class Solution(object):
    def removeDuplicates(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        j = 1
        for i in range(len(nums) - 1):
            if nums[i] != nums[i + 1]:
                nums[j] = nums[i + 1]
                j += 1
        return j
        

// OR (Using Sets)

class Solution(object):
    def removeDuplicates(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        k = len(nums)
        ans = list(set(nums))
        ans.sort()
        for i in range(len(ans)):
            nums[i] = ans[i]
        return len(ans)

File: LeetCodeProblems-CPP/26. Remove Duplicates from Sorted Array.cpp
#include<vector>
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int i=1;
        int j=1;
        int n = nums.size();
        while(j<n){
            if(nums[j] != nums[i-1]){
                nums[i] = nums[j];
                i++;
            }
            j++;
        }
        return i;
    }
};